solution-base/mongodb/patches/mongodb-sharded-add-configsvr-service.patch

```diff
@@ -14,64 +14,22 @@ index 04b183a2..0fe370f2 100644
    {{- if .Values.configsvr.external.replicasetName -}}
      {{- .Values.configsvr.external.replicasetName }}
 diff --git a/solution-base/mongodb/charts/mongodb-sharded/templates/config-server/config-server-service.yaml b/solution-base/mongodb/charts/mongodb-sharded/templates/config-server/config-server-service.yaml
-new file mode 100644
-index 00000000..fff5cea7
---- /dev/null
+index fad94092..825b0161 100644
+--- a/solution-base/mongodb/charts/mongodb-sharded/templates/config-server/config-server-service.yaml
```

Comment:
Actually this file is not part of the charts, so if someone deletes the associated yaml, then fetch all the charts, this patch file won't work, I think it's fine, for this one, to have the patch actually creating the whole file, like previously